environment_strelka.yml

```diff
@@ -0,0 +1,12 @@
+name: strelka
```

Comment:
How do you plan to go about activating two environments at once?

Reply:
We don't, this was a workaround thing from before singularity.